src/components/Forms/SuggestionBox.js

```diff
@@ -0,0 +1,192 @@
+  const [urgency, setUrgency] = useState(5);
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [validationMessage, setValidationMessage] = useState(null);
+  const [submitted, setSubmitted] = useState(false);
+
+  const theme = useTheme();
+
+  const handleSubmit = event => {
+    const toSubmit = {
+      topic,
+      description,
+      benifits,
+      urgency,
+      name,
+      email,
+    };
+
+    //Custom Validation
+    const valuesToValidate = Object.values(toSubmit);
+    //Validates all required fields (initiated with null) are filled
+    if (valuesToValidate.includes(null)) {
+      setValidationMessage('All fields with * are required.');
+      return;
+    }
+
+    //posts to Netlify intigration
+    fetch('/', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+      body: encode({ 'form-name': 'contact', toSubmit }),
+    })
+      .then(() => console.log('Success!'))
+      .catch(error => console.log(error));
+
+    event.preventDefault();
+    setSubmitted(true);
+  };
+
+  if (submitted) {
+    return (
+      <Text fontSize="2xl" textAlign="center">
+        Thank you for your suggestion!
+      </Text>
+    );
+  }
+
+  return (
+    <FormControl
+      width="100%"
+      maxWidth="1000px"
+      margin="0 auto 3rem"
+      padding="0 24px"
+      onKeyPress={event => {
+        if (event.key === 'Enter') {
+          handleSubmit(event);
+        }
+      }}
+      //netlify form handling
+      method="post"
+      data-netlify-honeypot="bot-field"
+      data-netlify="true"
+      name="contact"
+    >
+      {/* netlify form handling */}
+      <input type="hidden" name="bot-field" />
+      <input type="hidden" name="form-name" value="contact" />
+      {/* netlify form handling */}
+
+      <Text fontSize="xl" textAlign="center">
+        Suggestion Box
+      </Text>
+
+      {/* renders when form is submitted with validation errors */}
+      {validationMessage && <Text textAlign="center">{validationMessage}</Text>}
+
+      <Flex direction="column" margin={theme.spacing.base}>
+        <FormLabel isRequired htmlFor="topic">
+          Topic
+        </FormLabel>
+        <Input
+          value={topic}
+          id="topic"
+          type="text"
+          placeholder="e.g. Business, Functionality, New Feature"
+          onChange={event => setTopic(event.currentTarget.value)}
+        />
+      </Flex>
+
+      <Flex direction="column" margin={theme.spacing.base}>
+        <FormLabel isRequired htmlFor="textField">
+          Description
+        </FormLabel>
+        <Textarea
+          value={description}
+          id="description"
+          placeholder="Description"
+          onChange={event => setDescription(event.currentTarget.value)}
+        />
+      </Flex>
+
+      <Flex direction="column" margin={theme.spacing.base}>
+        <FormLabel isRequired htmlFor="benifits">
+          Benifits
```

Comment:
Misspelling here, should be `Benefits`. Consider updating this for the variable names and such as well :)